operator/pkg/central/reconciler/reconciler_test.go

```diff
@@ -0,0 +1,46 @@
+package reconciler
+
+import (
+	"testing"
+
+	pkgReconciler "github.com/operator-framework/helm-operator-plugins/pkg/reconciler"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAddSelectorOptionIfNeeded(t *testing.T) {
+	tests := []struct {
+		name     string
+		selector string
+		wantLen  int
+		wantErr  bool
+	}{
+		{
+			name:     "empty selector",
+			selector: "",
+			wantLen:  0,
+			wantErr:  false,
+		}, {
+			name:     "non-empty selector",
+			selector: "foo=bar",
+			wantLen:  1,
+			wantErr:  false,
+		}, {
+			name:     "invalid selector",
```

Comment:
Did you forgot pushing the StackRox version unit tests? 🤔

Reply:
what do you mean? Do you mean adding a test case to setup a central selector with a label key and value representing the stackrox version?

Reply:
Yes

Reply:
Because this logic doesn't really know anything about what kind of selector we are actually going to use. It's all going to be injected

The fleedshard sync would create the `Central` with the proper label, and also install the operator with the label selector environment variable.